### ISSUE TITLE
Eslint shows one warning in all ignored files

### ISSUE THREAD

**Issue by kumarharsh** [Microsoft/vscode-eslint#106]
Eslint shows this warning in all files which I've ignored via the .eslintignore:

```
File ignored because of a matching ignore pattern. Use --no-ignore to override.
```

I think that this warning should not be shown, as the cli doesn't show it either. Is this default warning intentional, or should it be fixed?

**Comment by dbaeumer** [comment]
Can't reproduce using the latest ESLint extension and ESLint version. Closing. @kumarharsh please ping if you can still reproduce.

**Comment by kumarharsh** [comment]
@dbaeumer I'm still getting it. One file where this always comes is the `.eslintrc.js` file in the workspace root.
![2016-09-15 2](https://cloud.githubusercontent.com/assets/760112/18530332/b6747b80-7aee-11e6-95be-ef7c71187632.png)

**Comment by dbaeumer** [comment]
@kumarharsh this looks like eslint behavior. Do you see the same message in the terminal ?
